### PR TITLE
Automate test case for 'wego add --url' flag

### PR DESCRIPTION
**Pull request by rokshana-b**
**What changed?**
- [ ] Added test case for `wego add app --url`
- [ ] Added `--branch` flag to `wego add app --dry-run` test case

**Why?**
To add test coverage for additional WeGO features

### CODE REVIEW

**Review by josecordaz — APPROVED** [review]
LGTM

**Review by jrryjcksn — APPROVED** [review]
LGTM. There's also a new app add mode (upstream helm repository) that we can only test with `--url`. So, this will be a good starting place for that too.

File: test/acceptance/test/add_tests.go [hunk 1]
```diff
@@ -279,18 +279,18 @@ var _ = Describe("Weave GitOps Add Tests", func() {
...
 		var session *gexec.Session
 		private := true
-		branchName := "main"
+		branchName := "test-branch"
 		appManifestFilePath := "./data/nginx.yaml"
 		defaultSshKeyPath := os.Getenv("HOME") + "/.ssh/id_rsa"
 		appRepoName := "wego-test-app-" + RandString(8)
 		workloadName := "nginx"
 		workloadNamespace := "my-nginx"
 		url := "ssh://git@github.com/weaveworks-gitops-test/" + appRepoName + ".git"
```
Comment (iahmad9) [inline test/acceptance/test/add_tests.go:292]: Should not hardcode the org part, instead should fetch from os.Getenv("GITHUB_ORG") environment variable.

File: test/acceptance/test/add_tests.go [hunk 2]
```diff
@@ -330,4 +334,52 @@ var _ = Describe("Weave GitOps Add Tests", func() {
 			verifyWegoAddCommandWithDryRun(appRepoName, WEGO_DEFAULT_NAMESPACE)
 		})
 	})
+
+	// Eventually this test run will include all the remaining un-automated `wego app add` flags.
+	It("SmokeTest - Verify 'wego app add' works when --url flag is specified", func() {
+		var repoAbsolutePath string
+		private := true
+		appRepoName := "wego-test-app-" + RandString(8)
+		url := "ssh://git@github.com/weaveworks-gitops-test/" + appRepoName + ".git"
```
Comment (iahmad9) [inline test/acceptance/test/add_tests.go:343]: Same as above => Should not hardcode the org part, instead should fetch from os.Getenv("GITHUB_ORG") environment variable.

**Review by iahmad9 — CHANGES_REQUESTED** [review]
Please change the hardcode GitHub org to env variable.

**Pull request merged**